AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -718,27 +718,20 @@ public void response(List<Map<String, Object>> requests, JSONObject response, in
     LeanplumInternal.triggerStartIssued();
   }
 
-  private static void handleApiResponse(JSONObject response, List<Map<String, Object>> requests,
```

Comment:
Is there a Jira ticket associated with this  PR?

Reply:
We have a Zendesk ticket where we reported the issue in November. See ticket #29522.